src/components/LaunchDetails/theme.ts

```diff
@@ -0,0 +1,14 @@
+import { createTheme, responsiveFontSizes } from '@mui/material/styles';
+
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: '#90caf9',
+      light: '#c3fdff',
+      dark: '#5d99c6',
+      contrastText: '#212121',
+    },
+  },
+});
```

Comment:
1. This theme should be in `app.tsx`
2. colors moved to `_variables.scss`

Reply:
1. Done